scripts/eventbrite-API.py

```diff
@@ -17,82 +17,121 @@
 
 base_url = "https://www.eventbriteapi.com/v3/"
 
-####
-# TODO: instead of passing a giant viewport for the whole US,
-# iterate through the states.
-# An approach that could work is to get the centroid for every
-# state and have a really wide "within" param.
-# There's going to be overlap between states with that approach,
-# but we can eliminate duplicates in the DB afterwards.
-####
-# us_states = ["AL", "AK", "AZ", "AR", "CA", "CO", "CT", "DE",
-#              "FL", "GA", "HI", "ID", "IL", "IN", "IA", "KS",
-#              "KY", "LA", "ME", "MD", "MA", "MI", "MN", "MS",
-#              "MO", "MT", "NE", "NV", "NH", "NJ", "NM", "NY",
-#              "NC", "ND", "OH", "OK", "OR", "PA", "RI", "SC",
-#              "SD", "TN", "TX", "UT", "VT", "VA", "WA", "WV", "WI", "WY"]
+us_states = [
```

Comment:
[SC]
1. Call the variable something more representative (`us_states_and_view_ports`)
2. Separate this data into separate file for neatness.

Reply:
Added SC's